org.eclipse.scanning.example/src/org/eclipse/scanning/example/detector/PosDetectorModel.java

```diff
@@ -0,0 +1,26 @@
+package org.eclipse.scanning.example.detector;
+
+import org.eclipse.scanning.api.device.models.AbstractDetectorModel;
+
+public class PosDetectorModel extends AbstractDetectorModel {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)